src/utils/constants.ts

```diff
@@ -231,8 +233,33 @@ type SocialAuthOption = {
   teamId?: string
   keyId?: string
   note?: string
+  provider?: string
 }
 
+const passkeyLogins: readonly SocialAuthOption[] = [
+  {
+    name: 'Passkey',
+    verifier: 'passkey',
+    icon: TelegramIcon,
```

Comment:
Why TelegramIcon? Replace it with Actual icon from Figma

Reply:
There is no icon for passkeys in Figma. So I removed it.